src/charts/LineChart/LineChart.tsx

```diff
@@ -64,16 +63,24 @@ export default function LineChart({
 
   // if no xKey datatype is passed in, determine if it's Date
   if (!xDataType) {
-    xDataType = inferXDataType(data[0], xKey);
+    xDataType = inferXDataType(data[0], xKey)
   }
 
-  const xAccessor: xAccessorFunc = xDataType === 'number' ? (d) => d[xKey] : (d) => new Date(d[xKey]);
-  const yAccessor: yAccessorFunc = (d) => d[yKey];
+  const xAccessor: AccessorFunc =
```

Comment:
There is different type definition for xAccessor and yAccessor as they return different values. Having one AccessorFunc  definition for both functions breaks code.